_posts/2023-07-05-trino-fest-2023-arcticwolf.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title: "AWS Athena (Trino) in the cybersecurity space"
+author: "Anas Shakra, Ryan Duan"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/ArcticWolf.png
+show_pagenav: false
+---
+
+AWS Athena has become a new and efficient service for companies to query their
+data using Trino. AWS Athena is a serverless, interactive analytics service
+built on open-source frameworks, supporting open table and file formats and
+providing a simplified, flexible way to analyze petabytes of data where it lives.
+Arctic Wolf Networks, a cybersecurity company that provides security monitoring
+to cyber threats, is one of these companies that have recently switched to using
+AWS Athena. Senior software developer Anas Shakra from Arctic Wolf Networks gave
+a talk at [Trino Fest 2023]({% post_url 2023-06-20-trino-fest-2023-recap %})
+detailing their switch to AWS Athena and how "queries that took hours with old
+solution now take around a minute today". Tune in to the talk or you can read
+the recap!
+
+<!--more-->
+
+{% youtube WCuJaW7zC8k %}
+
+## Recap
+
+Generally, data access use-cases fall under three categories: investigations,
```

Comment:
```suggestion
At Arctic Wolf, data access use-cases fall under three categories: investigations,
```
This isn't general, this is specific to what they do. 😛

Reply:
;-)